test-result-summary-client/src/Build/Summary/ResultSummary.jsx

```diff
@@ -59,47 +59,44 @@ export default class ResultSummary extends Component {
         let jdkImplOpts = [];
         builds.map((build, i) => {
             const buildName = build.buildName.toLowerCase();
-            const tokens = buildName.match(regex);
-            if (Array.isArray(tokens) && tokens.length > 5) {
-                const [_, jdkVersion, jdkImpl, level, group, platform] = tokens;
+            const {_, jdkVersion, jdkImpl, level, group, platform} = getInfoFromBuildName(buildName);
```

Comment:
We can get rid of `regex` since it's not used here anymore. Also, I think we need some check here as done with the if block before.

Reply:
yes, we need to check to make sure the return value is valid

Reply:
We need to get rid of `const regex = /^Test_openjdk(\w+)_(\w+)_(\w+).(.+?)_(.+?_.+?(_xl)?)(_.+)?$/i;` since it's no longer used here.